docs/dev/intermediate/custom-configs.md

```diff
@@ -11,79 +11,78 @@ description: An intermediate overview of how to implement custom configs for you
 :::
 
 ::: info
-This tutorial is taken and adapted from the [BepInEx Configuration Documentation](https://docs.bepinex.dev/articles/dev_guide/plugin_tutorial/4_configuration.html). For more resources refer to that.
+This tutorial is taken and adapted from the [BepInEx Configuration Documentation](https://docs.bepinex.dev/articles/dev_guide/plugin_tutorial/4_configuration.html).<br> 
+For more information refer to those docs.
 :::
 
-## Creating Config Entries
-Create a config class and add entries for any variables that you want to be configurable.
+## 1. Create the Config
+Begin by creating a `Config` class, then write out all the variables you want to be configurable.
 
 ```cs
 public class Config
 {
-    public static ConfigEntry<string> configGreeting;
-    public static ConfigEntry<bool> configDisplayGreeting;
+    public bool PLUGIN_ENABLED { get; private set; }
+    public float MOVEMENT_SPEED { get; private set; }
 
     // ...
 }
 ```
 
-Then we can start binding our config entries to the fields we just created inside of a class constructor.
+Implement the constructor with a parameter referencing the file that is created by **BepInEx**.<br>
+Then we can start binding our config entries to the properties we just created.
 
 ```cs
 public Config(ConfigFile cfg)
 {
-    configGreeting = cfg.Bind(
-            "General",                          // Config section
-            "GreetingText",                     // Key of this config
-            "Hello, world!",                    // Default value
-            "Greeting text upon game launch"    // Description
-    );
-    
-    configDisplayGreeting = cfg.Bind(
-            "General.Toggles",                  // Config subsection
-            "DisplayGreeting",                  // Key of this config
-            true,                               // Default value
-            "To show the greeting text"         // Description
-    );
+    PLUGIN_ENABLED = cfg.Bind(MyPluginInfo.GUID, "bPluginEnabled", true,
+        "Enable or disable the plugin globally."
+    ).Value;
+
+    MOVEMENT_SPEED = cfg.Bind(MyPluginInfo.GUID, "fMovementSpeed", 6.9f,
+        "The speed at which the local player moves."
+    ).Value;
```

Comment:
Doing this (.Value) does not support changing the config settings from any of the in-game config mods; please consider changing this back to allow changing config settings in-game via the config mods.